src/promdens/promdens.py

```diff
@@ -299,34 +322,24 @@ def calc_field(self, pulse: LaserPulse) -> None:
         :param pulse: LaserPulse dataclass containing laser pulse parameters (frequency, fwhm...)
         """
         self.pulse = pulse
+        # TODO: Remove all these extra assignments
```

Comment:
Going to do this in follow-up PR to keep this one small.

Reply:
Ok, cool.